src/kwlib.sh

```diff
@@ -178,8 +197,14 @@ function find_kernel_root()
 function get_kernel_release()
 {
   local flag="$1"
-  # TODO: Maybe we need to remove this error redirection
-  local cmd='make kernelrelease 2> /dev/null'
+  local cmd='make kernelrelease'
```

Comment:
I don't have a way to reproduce this bug in my current setup. Could you check if the flag `--silent` removes the requirement of this /dev/null? I mean:

```
make kernelrelease --silent
```

Reply:
It still there.
Here's .config file that I am using from ubuntu server 22.04.1
https://pastebin.com/NGNVzgE8
using kernel source v5.17 (any source from v5.13-v5.17 can reproduce this bug)